website/views/pages/homepage.ejs

```diff
@@ -283,7 +283,7 @@
           <a
             style="max-width: 220px"
             class="btn btn-block btn-md btn-primary mx-4"
-            href="/get-started"
+            href="/get-started" onClick="ga('send','event','[Fleetdm]','[Buttons]','[try it bottom]');"
```

Comment:
Same as https://github.com/fleetdm/fleet/pull/4474  (I think we should consolidate these PRs into 1.  @ksatter could you show @DrewBakerfdm how to do that?)

Reply:
@mikermcneil Definitely! Just let me know when, @DrewBakerfdm

Reply:
Also feel free to subscribe to each other’s calendars and just book whenever is open during working hours